composer.json

```diff
@@ -0,0 +1,47 @@
+{
+    "name": "diablomedia/gpcsv-generator",
+    "type": "library",
+    "description": "Library to assist with the generation of CashPro Global Payments GPCSV files",
+    "keywords": ["gpcsv"],
+    "homepage": "https://github.com/diablomedia/gpcsv-generator",
+    "license": "LGPL-3.0-or-later",
+    "authors": [
+        {
+            "name": "Ari Pringle",
+            "email": "ari@diablomedia.com",
+            "homepage": "https://diablomedia.com"
+        }
+    ],
+    "require": {
+        "ext-filter": "*",
+        "php": "^7.2"
+    },
+    "autoload": {
+        "psr-4": {
+            "GPCSV\\": "src/GPCSV"
+        }
+    },
+    "autoload-dev": {
+        "psr-4": {
+            "GPCSV\\Tests\\": "tests"
+        }
+    },
+    "require-dev": {
+        "phpunit/phpunit": "^8.5.5",
+        "diablomedia/phpunit-pretty-printer": "^4.0",
+        "phpstan/phpstan": "^0.12.28",
```

Comment:
Typically we've been locking these dependencies (phpstan/psalm/php-cs-fixer) and letting Dependabot bump them when a release comes out (this exposes any issues in the dependabot PR rather than in a CRON build or when going to make a small update and composer pulls down a new version of one of these libs).  I *think* dependabot will make PRs with this sort of version range, but that may depend on settings and if the `composer.lock` file is committed.